api/api_public_klay.go

```diff
@@ -70,3 +75,61 @@ func (s *PublicKlayAPI) Syncing() (interface{}, error) {
 		"knownStates":   hexutil.Uint64(progress.KnownStates),
 	}, nil
 }
+
+// EncodeAccountKey gets an account key of JSON format and returns RLP encoded bytes of the key.
+func (s *PublicKlayAPI) EncodeAccountKey(accKey accountkey.AccountKeyJSON) (hexutil.Bytes, error) {
+	buffer := new(bytes.Buffer)
+	if accKey.KeyType == nil {
```

Comment:
The type of `KeyType` is uint8...?

Reply:
It was `uint8`, but is changed to `*uint8` in this PR. 
https://github.com/klaytn/klaytn/blob/5ab75cecae9a6478456ae387a7b31787c490d685/blockchain/types/accountkey/account_key_serializer.go#L35-L38
You can see the same code in the middle part of this change.